### ISSUE TITLE
Direct video link (?raw=1) doesn't work for all video

### ISSUE THREAD

**Issue by poka-IT** [iv-org/invidious#3079]
For some video, direct link doesn't work.

For instance, this work: https://yewtu.be/embed/ZoeFQsQ6300?raw=1
But this doesn't work: https://yewtu.be/embed/nNKYkJn2Lts?raw=1

Do you know the reason ?

**Comment by unixfox** [comment]
Use https://yewtu.be/embed/nNKYkJn2Lts?raw=1&local=true
For forcing the proxy

**Comment by unixfox** [comment]
Probably because you are loading a video from another IP address, usually the music videos are encrypted by Youtube and are tied to the IP address. If you try loading it from another IP address you get a 403 error.

If you need to download a video you are better off using the `latest_version` endpoint like this: https://yewtu.be/latest_version?id=nNKYkJn2Lts&itag=18&local=true

**BUT don't try to circumvent the disability to download a video on an invidious instance**, if the download button is disabled it's for good purpose. Use another instance that allow download if you need that.

**Comment by poka-IT** [comment]
Thank you for this explanations, and yes it's just for music.

Your first answer seems to solve the problem for the url I exposed as an example, but not for all of them, for instance:
https://yewtu.be/embed/RLR4JQrHpY0?listen=1&raw=1&local=true

I found the itags list: https://gist.github.com/sidneys/7095afe4da4ae58694d128b1034e01e2
This is not for download but only for streaming from a Flutter app.

So it seems to work as expected with this kind of request:
https://yewtu.be/latest_version?id=09cZRYupO4s&itag=140&local=true&listen=1

Now I have TLS untrusted certificate issues from my app, but this is a complete differente problem.
Thank you for your help.
